src/locales/en/header.json

```diff
@@ -0,0 +1,5 @@
+{
+  "titles": ["Dorm room social manager", "LOGIN", "Create your account", "REGISTER"],
+  "buildingName": "IV DS",
+  "faculty": "Technical University Of Lodz"
+}
```

Comment:
this translation should not be in header component. We should have translations for `LoginPage` and `RegisterPage`

Reply:
Should be fixed in 7cb59ad